### PR TITLE
Comb Sort algorithm implementation

### PR DESCRIPTION
**Pull request by kavithajm**
image

### CODE REVIEW

**Comment by kavithajm** [comment]
Formatted the code using gofmt.

File: sort/sorts_test.go [hunk 1]
```diff
@@ -122,6 +122,10 @@ func TestSelection(t *testing.T) {
 	testFramework(t, SelectionSort)
 }
 
+func TestComb(t *testing.T) {
+	testFramework(t, combSort)
```
Comment (tjgurwara99) [inline sort/sorts_test.go:126]:
```suggestion
	testFramework(t, Comb)
```

File: sort/sorts_test.go [hunk 2]
```diff
@@ -216,6 +220,10 @@ func BenchmarkSelection(b *testing.B) {
 	benchmarkFramework(b, SelectionSort)
 }
 
+func BenchmarkComb(b *testing.B) {
+	benchmarkFramework(b, combSort)
```
Comment (tjgurwara99) [inline sort/sorts_test.go:224]:
```suggestion
	benchmarkFramework(b, Comb)
```

**Review by tjgurwara99 — CHANGES_REQUESTED** [review]
Please fix the errors in the test file.

**Review by tjgurwara99 — APPROVED** [review]

**Review by raklaptudirm — APPROVED** [review]

**Review by tjgurwara99 — APPROVED** [review]

**Comment by raklaptudirm** [comment]
@tjgurwara99 The workflow seems to be frozen.

**Comment by tjgurwara99** [comment]
> @tjgurwara99 The workflow seems to be frozen.

@raklaptudirm Yeah, it keeps happening when we update the branch. I can't seem to find out the reason for this. It gets fixed when there is more activity happening on the PR like commits etc but this freeze should not really happen. I will log a bug to the GitHub/runner repo to see if they can see where this issue is coming from, like If it is because of the changes I made recently or is its something else.

**Review by siriak — APPROVED** [review]

**Pull request merged**